lib/master.js

```diff
@@ -112,6 +118,20 @@ class Master extends BaseService {
             this._logger.log('warn/service-runner', 'startup finished');
             this._checkHeartbeat();
         });
+
+    }
+
+    _startMetricsEndpoint() {
+        HTTP.createServer((req, res) => {
+            AggregatorRegistry.clusterMetrics((err, metrics) => {
+                if (err) {
+                    this._logger.log('error/service-runner', err);
+                }
+                res.setHeader('Content-Type', AggregatorRegistry.contentType);
+                res.write(metrics);
+                res.end();
+            });
+        }).listen(this.config.metrics.prometheus.port);
```

Comment:
1. Nothing ever closes the server.
2. It would probably be nice to move this into a separate file.
3. What if prometheus is not configured - this will explode.

Reply:
Explosions are not fun.  Please see latest.  This was removed and moved into a separate file.